core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala

```diff
@@ -52,7 +52,7 @@ class RichPresentationCompilerThatNeedsJavaLibsSpec extends EnsimeSpec
 }
 
 class RichPresentationCompilerSpec extends EnsimeSpec
-    with IsolatedRichPresentationCompilerFixture
+    with SharedRichPresentationCompilerFixture
```

Comment:
we should perhaps go back to `Isolated` fixtures where possible. This was a performance hack that we may no longer need.